modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -61,13 +61,15 @@
     private ElmVersion elmVersion = ElmVersion.ELM_019;
 
     private static final String ELM_VERSION = "elmVersion";
+    private static final String PREFIX_CUSTOM_TYPE_VARIANTS = "prefixCustomTypeVariants";
```

Comment:
@trenneman Since the option is Elm specified, what about naming it as "elmPrefixCustomTypeVariants"?

Reply:
Good point, I'll update the PR.